sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/config/top-women-yearly/2023.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  title: 'Celebrating Leading Women In Higher Education',
+  subtitle: '<i>Diverse</i> presents 25 women making a difference in the world of academe.',
+  //createdBy: 'Compiled by Walter Hudson',
```

Comment:
This needs to be `// createdBy: 'Compiled by Walter Hudson',` or preferably just removed